app/models/user.rb

```diff
@@ -11,4 +11,11 @@ class User < ApplicationRecord
   has_secure_password
   validates :password, presence: true,
     length: {minimum: Settings.app.models.user.passwordvalidatelength}
+
+  # Returns the hash digest of the given string.
+  def self.digest string
+    cost = ActiveModel::SecurePassword.min_cost ? BCrypt::Engine::MIN_COST :
+      BCrypt::Engine.cost
```

Comment:
nếu đã sử dụng toán tử 3 ngôi thì không được ngắt dòng. Nên viết trên 1 dòng
Nếu ngắt dòng thì phải chuyển sang dùng if .. else